### PR TITLE
Feat grid

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
Added three grid layouts

- two column 20-80 layout 
- two column 50-50 layout
- three equal column layout

Preview: https://deploy-preview-26--hydrogen-ui.netlify.app/docs/grid

### CODE REVIEW

File: components/grid.css [hunk 1]
```diff
@@ -1,11 +1,11 @@
 .grid {
   display: grid;
+  min-height: 100px;
```
Comment (Jia-A) [inline components/grid.css:3]: px to rem, maybe you forgot to change it.

**Review by Jia-A — COMMENTED** [review]
Looks good to me.

File: components/grid.css (hunk 1 above)
Comment (junaidshaikh-js) [reply components/grid.css:3]: No, I just added it for the styling purpose, to show the demo.

Comment (Jia-A) [reply components/grid.css:3]: Ohh, then it's all cool. Merge it.

**Review by hinanshisuthar — COMMENTED** [review]
Looks good to me

**Pull request merged**